contents/blog/remote-first.md

```diff
@@ -0,0 +1,47 @@
+
+## Trust
+If you don't trust your employees, you shouldn't have hired them in the first place. At PostHog, we extend trust to our staff members in everything we do, from work output to spending company money. For example, we don't track hours - not only because it would be an administrative nightmare to try and reconcile submitted time-tracking reports, but also because output is what matters - not butts in seats. Secondly, leave days are permissionless at PostHog - this means that you can put in leave whenever you like without needing approval - as long as your workload is sorted for the period you're gone. This means that staff members can take some time off to rest, work on personal projects, or run errands as needed.
+
+Lastly, we give everyone at PostHog a company credit card with a set monthly limit that can be raised as needed. Purchases below a certain limit ($75 as of writing) don't need to be explained - staff members can make such purchases at their discretion. Any purchases larger than those amounts merely require a receipt and a memo to identify the purchase - but other than that, we don't micromanage staff spending. We trust your staff to use their privileges prudently, and we haven't had any issues so far with the initiative.
+
+## Iteration
+Perfection is the enemy of progress, and at PostHog we encourage rough takes published regularly over perfect work submitted sporadically. There are 2 main reasons for this: streamlining and quality assurance.
+
+Your pull request (the main way we make changes to our website and platform) might be required before another pull request can go through. It's thus important that everyone publishes regularly to relive any bottlenecks in the workflow.
+
+Secondly, publishing regularly makes it easier for other team members to spot mistakes and correct them early on. As an open-source product built on transparency, we extend this philosophy to our work output, too.
+
+Iteration is even more important for fast-growing remote teams, and technical teams can especially benefit from adopting an iterative approach to delivering work.
+
+## Diversity
+It's a no-brainer that diversity strengthens teams, and remote working allows you to deliberately hire people from different places and backgrounds. We've got team members in the US, the UK, Estonia, Congo, South Africa, the Ukraine, and other countries. We range all ages from 19 (our youngest dev) to 40-something (our business head, Charles). Some have advanced degrees in their field, others have degrees in fields totally unrelated to their current roles, and others are self-taught gurus who skipped or dopped out of college altogether. This cauldron of experiences and expertise generates better ideas that fortify our remote work model.
```

Comment:
Hah, I'm 32! We're not actually that diverse age-wise, I wouldn't shout about it.

Reply:
Had no idea - ok will omit that.